nixos/modules/services/x11/display-managers/lightdm-greeters/lomiri.nix

```diff
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  dmcfg = config.services.displayManager;
+  ldmcfg = config.services.xserver.displayManager.lightdm;
+  cfg = ldmcfg.greeters.lomiri;
+
+in
+{
+  meta.maintainers = lib.teams.lomiri.members;
+
+  options = {
+    services.xserver.displayManager.lightdm.greeters.lomiri = {
+      enable = lib.mkEnableOption (lib.mdDoc "lomiri's greeter as the lightdm greeter");
+    };
+  };
+
+  config = lib.mkIf (ldmcfg.enable && cfg.enable) {
```

Comment:
Note to self: I think this module is highly likely to be missing some stuff in case `services.xserver.displayManager.lightdm.greeters.lomiri.enable && (!services.desktopManager.lomiri.enable)`, basically requiring some code duplication (since the shell is also the greeter).

IMO that's *prolly* not worth blocking this on? I'm fine with a DE-specific DM being abit broken when not using said DE, will just put it on my TODO list.